src/Servers/Kestrel/Transport.Quic/test/QuicTestHelpers.cs

```diff
@@ -63,8 +63,12 @@ public static FeatureCollection CreateBindAsyncFeatures(bool clientCertificateRe
             sslServerAuthenticationOptions.RemoteCertificateValidationCallback = RemoteCertificateValidationCallback;
             sslServerAuthenticationOptions.ClientCertificateRequired = clientCertificateRequired;
 
+            var kestrelServerLimitsFeature = new KestrelServerLimitsFeature();
+            kestrelServerLimitsFeature.KeepAliveTimeout = new KestrelServerLimits().KeepAliveTimeout;
+
             var features = new FeatureCollection();
             features.Set(sslServerAuthenticationOptions);
+            features.Set(kestrelServerLimitsFeature);
```

Comment:
Where does this get set in real life outside of tests? Does this need to be set in TransportManager or something?